src/AppShed/Extensions/SpreadsheetBundle/Resources/config/services.xml

```diff
@@ -52,8 +52,30 @@
                 <argument>false</argument>
             </call>
         </service>
-        <service id="geoService"
-                 class="AppShed\Extensions\SpreadsheetBundle\Service\GeoService">
+        <service id="app_shed_extensions_spreadsheet.geo_client"
+                 class="GuzzleHttp\Client"
+                 factory-method="get"
+                 factory-class="AppShed\Extensions\SpreadsheetBundle\DependencyInjection\GuzzleClientFactory">
+            <argument type="collection">
+                <argument key="base_url" type="collection">
+                    <argument>http://maps.google.com/maps/api/geocode/json?sensor={sensor}</argument>
+                    <argument type="collection">
+                        <argument key="sensor">false</argument>
+                    </argument>
+                </argument>
+                <argument key="defaults" type="collection">
+                    <argument key="query" type="collection">
+                        <argument key="address"></argument>
```

Comment:
You don't need to set the address to be empty by default! what is the point?

You can use this to set the sensor to be false